### ISSUE TITLE
Remove the 'Load More' button from the Activity view and replace with automatic loading when the user scrolls down using a skeleton loading state

### ISSUE THREAD

**Issue by benjthayer** [status-im/status-desktop#8987]
In the current implementation of the Activity tab in the wallet, there is a load more button which enables the user to load activity further back in time. Speaking with @John-44, the plan is to replace this with automatic loading on scroll, removing the requirement for an explicit load more button.

The designs for the skeleton loading states for the Activity feed have been created here:
https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=10383%3A122033&t=BkPkWao1O0kMutnk-4

<img width="1188" alt="Loading --_ Scroll" src="https://user-images.githubusercontent.com/110033914/211038673-d28f8ceb-2825-4e09-8843-bd62e5d814de.png">

Speaking with @Khushboo-dev-cpp there was a question on whether activity would continue to load indefinitely until the very start of the user's txn activity is reached and loaded. I assume so, but tagging @John-44 to confirm.

**Comment by John-44** [comment]
@benjthayer @Khushboo-dev-cpp yes as a user scrolls down, activity should continue to load until there is no more activity to load.  In addition, when activity is loaded it should be stored (and then synced between the user's devices) so that it doesn't need to be fetched a 2nd time when the user goes to view the activity again.

**Comment by John-44** [comment]
@Khushboo-dev-cpp does the api we are calling for this activity return 'no more transactions to fetch' message (or something along those lines) when there is no more activity to be fetched?  If so, @benjthayer we probable need a message to be displayed at the bottom of the list for this state that says something along the lines of "You have reached the beginning of this activity history" or "No more activity history can be fetched" (wording tbd).

**Comment by Khushboo-dev-cpp** [comment]
@John-44 : Currently the api does not return no, more transactions to load, however, maybe this can be modified , will have to check feasibility maybe @alaibe knows already?

Also when you mentioned that it should load till there no more activity to load, does that mean if there are 100 transaction and we have loaded 25 for now, the user can scroll upto 75 more items? Does this mean we need to before hand how many total transactions are present for the particular address, afaik, I dont think that information is available to us from the go side.

**Comment by benjthayer** [comment]
Incase possible:
<img width="1188" alt="End" src="https://user-images.githubusercontent.com/110033914/211047495-59b959da-fc51-4294-9bd4-3bf958bc3f20.png">

**Comment by John-44** [comment]
> Also when you mentioned that it should load till there no more activity to load, does that mean if there are 100 transaction and we have loaded 25 for now, the user can scroll upto 75 more items? Does this mean we need to before hand how many total transactions are present for the particular address, afaik, I dont think that information is available to us from the go side.

@Khushboo-dev-cpp what I mean is when when the user scrolls down past the end of the already fetched activity, we should display the skeletal loading state while we fetch say the next 25 transactions.  If the user then subsequently scrolls further down, we should again display the skeletal loading state while we fetch say the next 25 transactions.  etc, etc...   

We shouldn't attempt to fetched all of the transactions, we should just fetch blocks of 25 transactions in response to the user triggering a fetch by scrolling down past the already fetched transactions.

If the user scrolls down past the already fetched transactions, and there are no more transaction that can be fetched (e.g. we request the next 25 transactions from the API, but only say 7 transactions are returned, because the user has reached the beginning of their transaction history) then we should display what @benjthayer has designed and is shown in his comment above.

Does that makes sense?

**Comment by Khushboo-dev-cpp** [comment]
Note that there is glitch when loading tx initially and I have raised a bug for that https://github.com/status-im/status-desktop/issues/9320
